apps/storybook/stories/ui-explorer/SyntaxHighlighter.js

```diff
@@ -209,24 +217,22 @@ export default class CopyableCode extends Component {
     document.body.removeChild(tmp);
     focus.focus();
 
-    this.setState({ copied: true }, () => {
-      window.setTimeout(() => this.setState({ copied: false }), 1500);
-    });
+    if (this._isMounted) {
```

Comment:
You shouldn't need this check -- onClick will run only when the component is mounted if the code is written correctly.

Reply:
This was the one that was actually throwing